app.rb

```diff
@@ -1,9 +1,25 @@
 require 'sinatra/base'
+require './lib/message'
+require './lib/database_connection_setup'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    erb(:index)
+  end
+
+  get '/messages' do
+    @messages = Message.all
+    erb(:messages)
+  end
+
+  post '/post-message' do
```

Comment:
Overall you've done a good attempt with this challenge. 

I would suggest having a look at RESTful routes. Here your post would changes to **post '/messages' do**


See below for more examples:

get '/peeps' -> is for listing out all peeps
post '/peeps' -> is for creating a peep
get '/peeps/1' -> is for viewing the peep with an id of 1
get '/peeps/new' -> would return the form for creating a new peep
get '/peeps/1/edit' -> would return a form for editing the peep with an id of 1
put '/peeps/1' -> would update the peed with an id of 1
delete '/peeps/1' -> would delete the peep with an id of 1

More info here: https://github.com/makersacademy/course/blob/main/pills/rest.md